apps/fz_http/lib/fz_http/events.ex

```diff
@@ -7,20 +7,28 @@ defmodule FzHttp.Events do
 
   # set_config is used because devices need to be re-evaluated in case a
   # device is added to a User that's not active.
-  def update_device(_device) do
+  def update_device(device) do
     GenServer.call(vpn_pid(), {:set_config, Devices.to_peer_list()})
+    GenServer.call(wall_pid(), {:add_rules, Rules.nftables_device_spec(device)})
```

Comment:
That'd be ideal, although I'm not sure if there is a mechanism already in Elixir to handle this, otherwise I think just swapping the order would be enough?

If the rules are added but the device is not, no harm done, the other way around the added device could bypass the rules for that user.

Reply:
Cool. Sounds good. Failing to add the rule (with `FzCommon.CLI.exec!`) will blow up and stop the addition of the device.